src/middlewared/middlewared/plugins/vm/vms.py

```diff
@@ -51,6 +51,7 @@ class VMModel(sa.Model):
     arch_type = sa.Column(sa.String(255), default=None, nullable=True)
     machine_type = sa.Column(sa.String(255), default=None, nullable=True)
     uuid = sa.Column(sa.String(255))
+    command_line_args = sa.Column(sa.JSON(), default=[], nullable=False)
```

Comment:
Why is it JSON, aren't arguments just a string? You can use `shlex.split` to split them into separate arguments then.